torch/autograd/saved_variable_default_hooks.py

```diff
@@ -5,3 +5,19 @@ def set_saved_tensors_default_hooks(pack_hook, unpack_hook):
 
 def reset_saved_tensors_default_hooks():
     torch._C._autograd._reset_default_hooks()
+
+def set_save_on_cpu_hooks():
```

Comment:
The PR body says this is supposed to be a context manager but it doesn't look like one

Reply:
Good catch, there should not be mentions of context managers anymore.